playbooks/roles/githubsshkeys/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+add_ssh_keys_from_github:
```

Comment:
@amirtds `add_ssh_keys_from_github` should go away and you should just have `githubsshkeys_usernames`.